README.md

```diff
@@ -1 +1 @@
-# Review-one-of-your-past-projects
\ No newline at end of file
+# vanel-julius-Miccroverse-trial
```

Comment:
-  update your readme so that it contains details about your project (e.g. the description, authors, copyright ). 👍

Reply:
You should let a student know why is it better to have Readme documentation. Please try to explain to the student why you are requesting this change.

Reply:
- update your readme so that it contains details about your project (e.g. the description, authors, copyright ). 👍
A README.md is the file that introduce the product to a user. It contains informations that explains why the product is useful and it is also ensures that the user knows how to use the product.